src/components/NurseryWithdrawals/index.tsx

```diff
@@ -12,9 +20,35 @@ type NurseryWithdrawalsProps = {
 };
 export default function NurseryWithdrawals({ reloadTracking }: NurseryWithdrawalsProps): JSX.Element {
   const trackingV2 = isEnabled('TrackingV2');
+  const [selectedPlantingSite, setSelectedPlantingSite] = useState<PlantingSite>();
+  const [plantsSitePreferences, setPlantsSitePreferences] = useState<Record<string, unknown>>();
+  const plantingSites = useAppSelector(selectPlantingSites);
+  const onSelect = useCallback((site: PlantingSite) => setSelectedPlantingSite(site), [setSelectedPlantingSite]);
+  const query = useQuery();
+  const onPreferences = useCallback(
+    (preferences: Record<string, unknown>) => setPlantsSitePreferences(preferences),
+    [setPlantsSitePreferences]
+  );
 
   if (trackingV2) {
-    return <NurseryPlantingsAndWithdrawals reloadTracking={reloadTracking} />;
+    return (
+      <PlantsPrimaryPage
+        title={strings.WITHDRAWALS}
+        onSelect={onSelect}
+        pagePath={APP_PATHS.NURSERY_WITHDRAWALS_V2}
+        lastVisitedPreferenceName='seedlings.withdrawals.lastVisitedPlantingSite'
```

Comment:
nursery.withdrawals.lastVisitedPlantingSite?

Reply:
Oh, I thought this preference name was set according to the position of the page in the menu. I can change it

Reply:
ah ok that's fine then, leave it as-is, i didn't remember